node/src/GlideClient.ts

```diff
@@ -310,4 +312,27 @@ export class GlideClient extends BaseClient {
     public time(): Promise<[string, string]> {
         return this.createWritePromise(createTime());
     }
+
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.
+     * The command will be routed to all primary nodes.
+     *
+     * See https://valkey.io/commands/flushall/ for more details.
+     *
+     * @param mode - The flushing mode, could be either {@link FlushMode#SYNC} or {@link FlushMode#ASYNC}.
```

Comment:
The syntax is slightly different for links
```suggestion
     * @param mode - The flushing mode, could be either {@link FlushMode.SYNC} or {@link FlushMode.ASYNC}.
```